### PR TITLE
Implement Wisp Super Staff Bros

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

**Comment by panpawn** [comment]
There's still things in here that are preventing it from being ready for merge, I'll work on it later.

**Comment by panpawn** [comment]
Okay, I'm done on this PR.  @jd4564  and other collabs can change it by pushing to the `wssb` branch.  I only wanted to create a PR where people could actually work on it from, not be the one to actually work on it.

File: mods/WSSB/scripts.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+				baseSignatureMove: 'gitgudfam', signatureMove: "Git Gud Fam",
+				evs: {hp:4, atk:252, spe:252}, nature: 'jolly',
+			},
+			'+Silveee': {
+				species: 'Absol-Mega', ability: 'Dank Aura', item: 'Black Glasses', gender: 'F',
+				moves: ['vcreate', 'extremespeed', 'boomburst'],
+				baseSignatureMove: 'serverbreak', signatureMove: "Server Break",
+				evs: {atk:252, spa:4, spe:252}, nature: 'naive',
+			},
+			' Mіdnight Prоf ΕХΟ': {
```
Comment (SilverTactic) [inline mods/WSSB/scripts.js]: what's with this random whitespace

File: mods/WSSB/scripts.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+				species: 'Umbreon', ability: 'Twilight Shield', item: 'Leftovers', gender: 'F', shiny: true,
+				moves: ['wish', 'curse', 'protect'],
+				baseSignatureMove: 'darkcausatum', signatureMove: "Dark Causatum",
+				evs: {hp: 252, spd: 252, def: 4}, nature: 'careful',
+			},
+			' Wаndo': {
```
Comment (SilverTactic) [inline mods/WSSB/scripts.js]: here too

**Comment by SilverTactic** [comment]
Since <code>mods/WSSB/statuses.js</code> doesn't actually implement anything new, I think it's okay to delete that file

**Comment by Itachi2000** [comment]
no travis errors
code seems preety legit
origin used the same code

**Comment by Itachi2000** [comment]
preety cool @Volcos

**Comment by panpawn** [comment]
There's no travis errors because jd and I took it over

**Comment by panpawn** [comment]
This no longer seems to be "out right crashing" - at least, not with the limited amount of testing that I had done on it so far today.

The fact of the matter is now we have two viable options: 1. We create bots to stress-test the tier, making sure that the rest of it is good to go and patch if/when necessary before merging. 2. We just push it to the production server and patch bugs if/as they happen.  

Of course, one option is definitely preferred, but the people who should be working on this aren't going to be, so we need to look at those two options with that in mind.

Also, in testing, there was one request before pushing

> LotusChamp Torpid: can you remove thick fat from alan?

So, what now, @jd4564?

**Comment by Tailzz** [comment]
Yᵒᵘ Oᶰˡʸ Lᶤᵛᵉ Oᶰᶜᵉ

**Comment by panpawn** [comment]
best commit summary ever

**Pull request merged**